main.js

```diff
@@ -166,30 +208,48 @@ define(function (require, exports, module) {
             }
         }
     }
-    
+
     function _documentChange(e) {
         _loadDoc(e.target, true);
     }
-    
+
     function _resizeIframe() {
         if (visible && $iframe) {
             var iframeWidth = panel.$panel.innerWidth();
             $iframe.attr("width", iframeWidth + "px");
         }
     }
-    
+
     function _updateSettings() {
         // Format
         var useGFM = _prefs.get("useGFM");
+
+        var renderer = new marked.Renderer();
+        if (_prefs.get('useGFMCheckboxes')) {
+            renderer.listitem = function(text) {
+                if (/^\[ \] \s*/.test(text)) {
+                    text = '<input type="checkbox" class="gfm-checkbox"> ' + text.substr(4);
+                } else if (/^\[x\] \s*/.test(text)) {
```

Comment:
This should be case insensitive, so `[x]` or `[X]` can be used to check an item.